scripts/cluster-upgrade.sh

```diff
@@ -297,11 +297,13 @@ do
             OCM_NAME=$(oc -n $CD_NAMESPACE get clusterdeployment $CD_NAME -o json | jq -r '.metadata.labels["api.openshift.com/name"]')
             for CLUSTER_NAME in $CLUSTER_NAMES
             do
-              if [ "$OCM_NAME" == "$(echo $CLUSTER_NAME | cut -c -15)" ];
```

Comment:
Why remove the cut?

Reply:
value of `OCM_NAME` is the full name, as supplied by the customer.  It's captured in a label `api.openshift.com/name`.  Ideally we move away from this name mess and go to the ID, it's getting to be a mess.  But without reworking a bunch of things (including upgrade process we're developing) I am sticking with the name.